lib/gateway/Shard.js

```diff
@@ -669,12 +669,13 @@ class Shard extends EventEmitter {
                 if(packet.d.user.username !== undefined) {
                     let user = this.client.users.get(packet.d.user.id);
                     let oldUser = null;
-                    if(user && (user.username !== packet.d.user.username || user.discriminator !== packet.d.user.discriminator || user.avatar !== packet.d.user.avatar || user.globalName !== packet.d.user.global_name)) {
+                    if(user && (user.username !== packet.d.user.username || user.discriminator !== packet.d.user.discriminator || user.avatar !== packet.d.user.avatar || user.globalName !== packet.d.user.global_name || user.avatarDecoration !== packet.d.user.avatar_decoration)) {
                         oldUser = {
                             username: user.username,
                             discriminator: user.discriminator,
                             avatar: user.avatar,
-                            globalName: user.globalName
+                            globalName: user.globalName,
+                            avatar_decoration: user.avatarDecoration
```

Comment:
`avatarDecoration` -> `avatarDecorationData`
```suggestion
                            avatarDecorationData: user.avatarDecorationData
```